File: programmers/java/16. 나누어 떨어지는 숫자 배열.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        int [] answer = {};
        
        answer = Arrays.stream(arr).filter(e -> e % divisor == 0).toArray(); 
        // 예외처리
        Arrays.sort(answer);
        
        if(answer.length == 0) {
            answer = new int[]{-1};
        }
        
        return answer;
    }
}

File: programmers/swift/16. 나누어 떨어지는 숫자 배열.swift
func solution(_ arr:[Int], _ divisor:Int) -> [Int] {
    
    let check = arr.filter { ($0)%divisor == 0 }
    print(check) //	[5, 10]
    
    return check.count == 0 ? [-1] : check.sorted()
}



// let cast = ["Vivien", "Marlon", "Kim", "Karl"]
// let lowercaseNames = cast.map { $0.lowercased() }
// // 'lowercaseNames' == ["vivien", "marlon", "kim", "karl"]
// let letterCounts = cast.map { $0.count }
// // 'letterCounts' == [6, 6, 3, 4

// let cast = ["Vivien", "Marlon", "Kim", "Karl"]
// let shortNames = cast.filter { $0.count < 5 }
// print(shortNames)
// // Prints "["Kim", "Karl"]"
